cinema/serializers.py

```diff
@@ -0,0 +1,21 @@
+from rest_framework import serializers
+
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(max_length=255)
+    description = serializers.CharField(max_length=255)
+    duration = serializers.IntegerField()
+
+    def create(self, validated_data):
+        movie = Movie.objects.create(**validated_data)
+        return movie
+
+    def update(self, instance, validated_data):
+        instance.title = validated_data.get("title", instance.title)
+        instance.description = validated_data.get("description", instance.description)
+        instance.duration = validated_data.get("duration", instance.duration)
+        instance.save()
+        return instance
```

Comment:
The create and update methods are correctly implemented, ensuring that the Movie model instances are created and updated with the validated data.